client/my-sites/stats/activity-log/index.jsx

```diff
@@ -32,22 +32,46 @@ import QueryRewindStatus from 'components/data/query-rewind-status';
 import QueryActivityLog from 'components/data/query-activity-log';
 import DatePicker from 'my-sites/stats/stats-date-picker';
 import StatsPeriodNavigation from 'my-sites/stats/stats-period-navigation';
-import { recordGoogleEvent } from 'state/analytics/actions';
+import { recordGoogleEvent } from 'state/analytics/actions';
 import ActivityLogRewindToggle from './activity-log-rewind-toggle';
-import { isRewindActive as isRewindActiveSelector } from 'state/selectors';
 import { rewindRestore as rewindRestoreAction } from 'state/activity-log/actions';
+import {
+	getRestoreProgress,
+	isRewindActive as isRewindActiveSelector,
+} from 'state/selectors';
 
 const debug = debugFactory( 'calypso:activity-log' );
 
 class ActivityLog extends Component {
 	static propTypes = {
 		isJetpack: PropTypes.bool,
-		siteId: PropTypes.number,
-		slug: PropTypes.string,
+		restoreProgress: PropTypes.shape( {
+			errorCode: PropTypes.string.isRequired,
+			failureReason: PropTypes.string.isRequired,
+			message: PropTypes.string.isRequired,
+			percent: PropTypes.number.isRequired,
+			restoreId: PropTypes.number.isRequired,
```

Comment:
Oh yeah, we don't know `restoreId` when we make the initial request, only after it's returned. Removed `isRequired` in f2a469306